Cabal/src/Distribution/Simple/PackageDescription.hs

```diff
@@ -0,0 +1,74 @@
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  Distribution.PackageDescription.Parsec
+-- Copyright   :  Isaac Jones 2003-2005
+-- License     :  BSD3
+--
+-- Maintainer  :  cabal-devel@haskell.org
+-- Portability :  portable
+--
+-- This defines parsers for the @.cabal@ format
+
+module Distribution.Simple.PackageDescription (
```

Comment:
the `Module` haddock is inconsistent with the package name. Also why was this moved from `Cabal-syntax`'s `Distribution.PackageDescription.Parsec`?

Reply:
> Also why was this moved from Cabal-syntax's Distribution.PackageDescription.Parsec?

Ah I see, `Verbosity` is in cabal-install

Reply:
Yeah, these are mostly just error-handling wrappers around the actual parsers. And since they use `Verbosity` it felt better to keep them in `cabal-install` instead of letting that leak into `Cabal-syntax`.